doc/source/install.txt

```diff
@@ -42,40 +41,38 @@ them, scikit-image should already be installed.
 
 If you prefer the regular Python distribution from python.org_, you can
 install scikit-image manually by downloading packages. You will need numpy_,
-scipy_ and the scikit-image package. You can find the packages in `Cristoph
-Gohlke's`_ web page with compiled Python packages. Here is the direct link to
+scipy_, and the scikit-image package. You can find the packages in `Cristoph
+Gohlke's`_ web page with compiled Python packages. Here are direct links to
 the `scipy section`_, `numpy section`_ and `scikit-image section`_. Make sure
-you download the right version for your system. E.g. numpy for Python 3.4 64
+you download the right version for your system. E.g., numpy for Python 3.4 64
 bit would be ``numpy‑1.9.2+mkl‑cp34‑none‑win_amd64.whl``.
 
 To install Goehlke's packages, use pip::
 
   pip install wheel
-  pip install --find-links Downloads scikit-image
+  pip install --find-links path/to/downloaded/packages scikit-image
```

Comment:
Forward slash is understood in Python on all operating systems, to the best of my knowledge, and pip should parse it correctly.